src/components/common/Application.tsx

```diff
@@ -0,0 +1,208 @@
+      name: "describeYourself",
+      type: "text",
+      value: formData.describeYourself,
+    },
+    {
+      heading: "Which year did you start taking Bitcoin seriously?",
+      name: "year",
+      type: "text",
+      value: formData.year,
+    },
+    {
+      heading: "Why is Bitcoin important to you?",
+      name: "background",
+      type: "text",
+      value: formData.background,
+    },
+    {
+      heading: "Portfolio/Github/Side-project link:",
+      name: "github",
+      type: "text",
+      value: formData.github,
+    },
+    {
+      heading: "How many hours per week are you willing to dedicate to this cohort?",
+      name: "time",
+      type: "text",
+      value: formData.time,
+    },
+    {
+      heading: "What do you hope to achieve through this Cohort?",
+      name: "why",
+      type: "text",
+      value: formData.why,
+    },
+  ];
+
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    setFormData((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      await axios.post("https://bot.bitshala.org/register", formData);
+      setSubmitted(true);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  return (
+    <>
+    {isRegOpen? (
+      <section className="grid place-items-center my-10 ">
+      <h3 className="flex h-14 cursor-pointer items-center font-bold lg:text-4xl">
+        <span className="text-orange px-2">Register</span> for the cohort now!
+      </h3>
+      {
+        submitted ? (
+          <Alert severity="success" className="my-5">
+            Your application was submitted successfully.
+          </Alert>
+        ) : (
+          <form
+            onSubmit={handleSubmit}
+            className="flex flex-col px-5 pt-2 gap-1 rounded-lg lg:w-1/2"
+          >
+            {
+              list.map((item) => {
+                return (
+                  <div key={item.name}>
+                    <p>{item.heading}</p>
+                    <Input
+                      className="border font-base text-sm rounded-lg block w-full p-2.5 mb-3"
+                      type={item.type}
+                      name={item.name}
+                      value={item.value}
+                      onChange={handleChange}
+                    />
+                  </div>
+                )
+              })
+            }
+
+            <p>Please select list of skills</p>
+            <Autocomplete
+              className="border font-base text-sm rounded-lg block w-full  mb-3"
+              multiple
+              options={skills}
+              value={formData.skills}
+              onChange={(e, value) => {
+                setFormData(prevState => ({
+                    ...prevState,
+                    skills: value 
```

Comment:
Type of `value` is `string[]`. This doesnt match the type of skills. Can you fix the type issue?